user/wpaperd/template.py

```diff
@@ -25,7 +25,15 @@ def post_build(self):
 
 
 def install(self):
-    self.install_bin(f"target/{self.profile().triplet}/release/wpaperd")
-    self.install_bin(f"target/{self.profile().triplet}/release/wpaperctl")
     self.install_license("LICENSE.md")
     self.install_man("wpaperd-output.5")
+    with self.pushd(f"target/{self.profile().triplet}/release"):
+        self.install_bin("wpaperd")
+        self.install_bin("wpaperctl")
+        with self.pushd("completions"):
+            self.install_completion("wpaperd.bash", "bash")
+            self.install_completion("wpaperctl.bash", "bash")
+            self.install_completion("_wpaperd", "zsh")
+            self.install_completion("_wpaperctl", "zsh")
+            self.install_completion("wpaperd.fish", "fish")
+            self.install_completion("wpaperctl.fish", "fish")
```

Comment:
you have to pass "wpaperctl" as the third arg for those since the default name is the pkgname, which means this currently only installs a wpaperd-named completion

Reply:
oh, fair